### PR TITLE
task_10 completed

### PR DESCRIPTION
**Pull request by Orleanskij**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,10 +26,34 @@
...
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.8.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <version>5.8.2</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (LikBez123) [inline pom.xml]: remove junit-vintage-engine dependency

File: src/test/java/CartTests.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public void beforeTest() {
+        cart = new Cart("andrew-cart");
+        car = new RealItem();
+    }
+
+    @Test
+    public void testPriceCalculationWhenAddingItem() {
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
```
Comment (LikBez123) [inline src/test/java/CartTests.java:18]: you can move this common logic into  @BeforeEach method

File: src/test/java/CartTests.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import org.junit.jupiter.api.Assertions;
+import shop.Cart;
+import shop.RealItem;
+
+public class CartTests {
+
+    Cart cart;
+    RealItem car;
+
+    @Before
```
Comment (LikBez123) [inline src/test/java/CartTests.java]: it is better to create new test data before each test. Please use here @BeforeEach annotation

File: src/test/java/JsonParserTests.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    @Before
+    public void beforeTests() {
+        file = new File("src/main/resources/" + cartName + ".json");
+    }
+
+    @Disabled("Disabled")
+    @Test
+    public void readFromFileTest() {
+        Parser parser = new JsonParser();
```
Comment (LikBez123) [inline src/test/java/JsonParserTests.java]: move the logic common to all tests to @BeforeEach method

File: src/test/java/JsonParserTests.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+public class JsonParserTests {
+
+    private Cart cart;
+    private File file;
+    private final String cartName = "andrew-cart2";
+    public static final String EXPECTED_CART_RESULT ="{\"cartName\":\"andrew-cart2\",\"realItems\":[],\"virtualItems\":[],\"total\":0.0}";
+
+
+    @Before
```
Comment (LikBez123) [inline src/test/java/JsonParserTests.java]: @BeforeEach

File: src/test/java/RealItemTest.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import shop.RealItem;
+
+public class RealItemTest {
+
+    @Test
+    public void TestRealItemWeight() {
```
Comment (LikBez123) [inline src/test/java/RealItemTest.java]: start your test method with lowercase

**Review by LikBez123 — CHANGES_REQUESTED** [review]

File: src/test/java/RealItemTest.java (hunk 6 above)
Comment (Orleanskij) [reply src/test/java/RealItemTest.java]: Fixed

File: src/test/java/JsonParserTests.java (hunk 5 above)
Comment (Orleanskij) [reply src/test/java/JsonParserTests.java]: Fixed

File: src/test/java/JsonParserTests.java (hunk 4 above)
Comment (Orleanskij) [reply src/test/java/JsonParserTests.java]: Fixed

File: src/test/java/CartTests.java (hunk 3 above)
Comment (Orleanskij) [reply src/test/java/CartTests.java]: Fixed

File: src/test/java/CartTests.java (hunk 2 above)
Comment (Orleanskij) [reply src/test/java/CartTests.java:18]: Fixed

File: pom.xml (hunk 1 above)
Comment (Orleanskij) [reply pom.xml]: Fixed

**Review by LikBez123 — APPROVED** [review]
Move common strings values to constants

**Pull request merged**